src/hooks/use-refresh.hook.ts

```diff
@@ -9,9 +9,9 @@ type UseHookRefresh = [
 export const useHookRefresh = (): UseHookRefresh => {
   const [internalFlag, setInternalFlag] = useState(false)
 
```

Comment:
Can you also write a TODO that 
this file is not used.

Reply:
Handled in [718ff20](https://github.com/ajktown/wordnote/pull/158/commits/718ff206cc67382bbdc1ccb45fd89aff91953879)